apps/src/templates/teacherDashboard/teacherSectionsRedux.ts

```diff
@@ -802,11 +802,22 @@ export const asyncLoadTeacherHomepageSectionData =
       ).then(response => dispatch(setCourseOfferings(response.value))),
       HttpClient.fetchJson<ParticipantTypesResponse>(
         '/dashboardapi/sections/available_participant_types'
-      ).then(response =>
-        dispatch(
+      ).then(response => {
```

Comment:
This is a fix to a bug where if you open the create section dialog before the available participant types was done loading, it would not auto-select the only available participant type.

This was messing with my tests, but has been a bug on Prod for a while and I finally got around to fixing it.